### ISSUE TITLE
train problem without OHEM

### ISSUE THREAD

**Issue by go2star** [msracver/Deep-Feature-Flow#40]
HI,  
        when I set  "ENABLE_OHEM:false" in the .yaml, the training goes gradient explosion.  But  I can reproduce the perfomance reported with "ENABLE_OHEM:true" .
        Anyone knows the reason, or how to fix it?

**Comment by go2star** [comment]
problem solved, just set BATCH_ROIS：-1 to BATCH_ROIS：128
but I wonder if I can still set BATCH_ROIS：-1, but change the code:
`bbox_loss = mx.sym.MakeLoss(name='bbox_loss', data=bbox_loss_, grad_scale=1.0 / cfg.TRAIN.BATCH_ROIS)`
to 
`bbox_loss = mx.sym.MakeLoss(name='bbox_loss', data=bbox_loss_, grad_scale=1.0 / len(rois))`
